README.md

```diff
@@ -203,6 +203,22 @@ import { setup } from 'goober';
 setup(React.createElement);
 ```
 
+With theme:
+```js
+import React from 'react';
+import { setup, styled } from 'goober';
+
+const theme = { primary: 'blue' };
+const ThemeContext = createContext(theme);
+const useTheme = () => useContext(ThemeContext);
+
+setup(React.createElement, undefined, useTheme);
+
+const ContainerWithTheme = styled('div')`
+  color: ${(props) => props.theme.primary};
+`;
+```
+
```

Comment:
This is really awesome @devrnt! Thank you for opening this one and fix it! 🎉 

Was about to suggest do add the `With theme` as a nested title so it shows in the TOC at the top, but then notice that the `With prefixer` section is missing as well. Do you wanna tackle the with prefixer one as well? This is totally approved by the way! 👍

Reply:
I'll open up a new PR with the nested `With theme` title. Can you point to the `With prefixer` section, I can't seem to find what you're talking about 🤔

Reply:
Awesome!

Yeah, sorry if the above is confusing. I've opened #191 for it as well. So your entry for the _theme_ param is really awesome, but that made me realise that another for _prefixer_ does not exist and it should. Inside here https://github.com/cristianbote/goober#setuppragma-function-prefixer-function-theme-function.